pr_sample_files/stakwork_sphinx-relay_app_63.ts

```
@@ -0,0 +1,199 @@
+    } else {
+      app.post('/unlock', async function (req: Req, res) {
+        const ok = await unlocker(req, res)
+        if (ok) {
+          sphinxLogger.info('=> relay unlocked!')
+          controllers.set(app)
+          socket.connect(server)
+          resolve(app)
+        }
+      })
+    }
+  })
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)